executor/point_get_test.go

```diff
@@ -171,28 +171,28 @@ func (s *testPointGetSuite) TestIndexLookupCharPK(c *C) {
 
 	// Test truncate without sql mode `PAD_CHAR_TO_FULL_LENGTH`.
 	tk.MustExec(`set @@sql_mode="";`)
-	tk.MustIndexLookup(`select * from t tmp where a = "aa";`).Check(testkit.Rows(`aa bb`))
```

Comment:
As @winoros suggested, It's better to also test the behavior of index lookup reader. I think you can:
1. use this query to test index lookup reader: `select * from t tmp use index(primary) where a = "aa";`
2. use this query to test point get plan: `select * from t tmp where a = "aa";`

Reply:
Cannot use `PointGet` plan if there is index hit in the query, even if the index hit is primary index?

Reply:
No, it can't. We can support this feature in the future. For now, it's just a workaround to make it using index lookup reader. Maybe we need a SQL Hint or SQL Mode to control whether to chose PointGetPlan or IndexLookupReader. /cc @eurekaka

Reply:
The index hints will be ignored when trying point get plan:
```mysql> create table t(a int primary key, b char(2));
Query OK, 0 rows affected (0.01 sec)

mysql> explain select * from t use index(primary) where a = 1;
+-------------+-------+------+-------------------+
| id          | count | task | operator info     |
+-------------+-------+------+-------------------+
| Point_Get_1 | 1.00  | root | table:t, handle:1 |
+-------------+-------+------+-------------------+
1 row in set (0.00 sec)
```
So I test index lookup by creating an index explicitly instead of providing an index hint, as you can see in the file `point_get_test.go`

Reply:
PS. The tested TiDB was built from the master branch

Reply:
Got it.